### ISSUE TITLE
EAN13 bbox out of bounds after bwipp change

### ISSUE THREAD

**Issue by nicohein** [adamchainz/treepoem#495]
### Python Version

3.9.6

### Package Version

3.22.0

### Description

Hi @adamchainz ,

after this bwipp commit https://github.com/bwipp/postscriptbarcode/commit/b4ae9e1bf1ea13c8ff1734e6d096b48222d0c429 and the changes made to the [ean13.ps line 59](https://github.com/bwipp/postscriptbarcode/blame/master/src/ean13.ps#L59C5-L59C25) the first diget is plotted out of bounds.

Here is an example (see the 4 at the very beginning):

![4260661767605_8204](https://github.com/adamchainz/treepoem/assets/6959110/671752d2-78c7-4849-9bf0-4d154076b99e)

Before the change:

![4260661767605_8204](https://github.com/adamchainz/treepoem/assets/6959110/00ff5d98-ac78-4706-a066-bf24b176aa40)

So we can see that the initial character was shifted left, however it now results in a position negative to the coordinate roots.

A very crude hack to solve the problem temporarily for EAN13 is to fix bbox boundaries manually in `def _get_bbox` resulting in:

![4260661767605_8204](https://github.com/adamchainz/treepoem/assets/6959110/236d9914-8cc8-4581-b767-100d62426acb)

Note, that due to the bwipp change the distance between the initial number and the first bar increased slightly.

Just for your interest, the temp hack for ean13 only:
```
    bounding_box_pattern = r"(%%BoundingBox:) ([0-9]+) ([0-9]+ [0-9]+ [0-9]+.*)"
    bounding_box_replacement = r"\1 " + f"-{scale} " + r"\3"
    err_output = re.sub(
        bounding_box_pattern, 
        bounding_box_replacement, 
        err_output, 
        count=1
    )

    bounding_box_pattern = r"(%%HiResBoundingBox:) ([0-9]+.[0-9]{6}) ([0-9]+.[0-9]{6} [0-9]+.[0-9]{6} [0-9]+.[0-9]{6})"
    bounding_box_replacement =  r"\1 " + f"-{scale:.6f} " + r"\3"
    err_output = re.sub(
        bounding_box_pattern, 
        bounding_box_replacement, 
        err_output, 
        count=1
    )
```

I am not sure if this requires just an additional ghostscript flag or a fix to bwipp. Please let me know your thoughts.

(I have ghostscript version 10.01.2 installed.)

Best
Nico

**Comment by adamchainz** [comment]
Please report the bug to bwipp, I don’t think we should add a workaround in treepoem.

**Comment by adamchainz** [comment]
Fixed in #496.
